knowledge-base/the-styles-of-the-web-report-viewer-s-toolbar-are-messed-up.md

```diff
@@ -0,0 +1,55 @@
+---
+title: The styles of the web report viewer's toolbar are messed up.
+description: The styles of the web report viewer's toolbar do not show up as expected.
+type: troubleshooting
+page_title: The styles of the web report viewer's toolbar are incorrect.
```

Comment:
You can also add the troubleshooting keyword here as the KB explains how to troubleshoot what is causing the issue. For example, "Troubleshooting what causes incorrect styling of the toolbar of the web report viewer"